### DISCUSSION TITLE
Welcome to rsiena Discussions!

### DISCUSSION THREAD

**Discussion by jhollway** [stocnet/rsiena#15] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hpg-hub** [comment]
Hi there, I also raised a question. Unfortunately, it has not been answered yet. How can I kickstart my discussion thread? ☺️
